gensrc/proto/doris_internal_service.proto

```diff
@@ -66,4 +66,11 @@ service PBackendService {
 
     // NOTE(murphy) check if it's really necessary
     rpc submit_mv_maintenance_task(starrocks.PMVMaintenanceTaskRequest) returns (starrocks.PMVMaintenanceTaskResult);
+
+    // local tablet reader
+    rpc local_tablet_reader_open(starrocks.PTabletReaderOpenRequest) returns (starrocks.PTabletReaderOpenResult);
```

Comment:
What if opened and not closed? 
If there any configuration about timeout or session ?

Reply:
There will be a localtabletmanager to manage all opened readers and do the GC work. yes we may need to add more params like timeout/sessionid(that's a larger scope, and we haven't discussed it yet) so maybe not included in this PR?